test/assets/wallet0.json

```diff
@@ -23,7 +22,8 @@
    "private": "bc484a9d6056138675fd8f24e5c1bdda649c7acb3fcfbc9552fffcf14a20023f",
    "chain": "6544c8c43d875da4b3bb9871d0d2d1b8253dc6a1e2c1666bca865fe18543828b",
    "address": "0x9988685Fc1E5c4079AaEA6A405e0f28956b1bbBe"
-  }
+  },
+  "derivationPath": "m/44'/60'/0'/0/0"
```

Comment:
Note that the derivation path is now per-wallet, not under `meta`. Breaking change.